### PR TITLE
If the tenant is not passed in the options no default value is used.

### PR DESCRIPTION
**Pull request by jkremser**
cc @abonas

### CODE REVIEW

File: lib/hawkular/base_client.rb [hunk 1]
```diff
@@ -146,14 +146,8 @@ def token_header
...
-      if @options[:tenant].nil?
-        { 'Hawkular-Tenant' => 'hawkular' }
-      elsif @options[:tenant].empty?
-        {}
-      else
-        { :'Hawkular-Tenant' => @options[:tenant],
-          'tenantId'         => @options[:tenant] }
-      end
+      @options[:tenant].nil? ? {} : { :'Hawkular-Tenant' => @options[:tenant],
+                                      'tenantId'         => @options[:tenant] }
```
Comment (jpkrohling) [inline lib/hawkular/base_client.rb]: Is there a place that documents where this `tenantId` is used? Also, why is the first a symbol and the second a string?

Comment (abonas) [inline lib/hawkular/base_client.rb]: isn't here the same information sent twice (2 keys with same value)? what am I missing?

**Comment by jkremser** [comment]
this PR is basically reverting the state to the original state before PR (https://github.com/hawkular/hawkular-client-ruby/pull/67)

I am not the author of the original code => no idea why 'tenantId' is there twice, but I left it there to be safe.

> Is there a place that documents where this tenantId is used?

I am not aware of it, but it'd be useful to have something. @jpkrohling do you think it's safe to remove the second occurrence? `'Hawkular-Tenant'` is 100% correct, but I am not aware of any component that needs also `'tenantId'` header.

**Comment by jpkrohling** [comment]
It seems it's there since the first version from Libor: https://github.com/hawkular/hawkular-client-ruby/blob/73848c5f611ea31675483f9d92a10a57ee64cb41/lib/hawkularclient.rb#L107 . It says to remove it once HWKMETRICS-187 is fixed , which seems to be the case.

**Comment by jkremser** [comment]
> It seems it's there since the first version from Libor

I knew it's either Libor's or vrockai's fault :)

**Comment by pilhuhn** [comment]
As HWKMETRICS-187 says it was fixed in 0.7, so with metrics 0.7+ we are indeed save to completely leave out the 'tenantId' header

**Pull request merged**